osquery/sql/virtual_table.cpp

```diff
@@ -28,6 +28,11 @@ FLAG(uint64,
      0,
      "Add an optional microsecond delay between table scans");
 
+FLAG(bool,
+     compat_index_all_extension_columns,
```

Comment:
I recommend having this as a hidden flag, and calling it `extensions_default_index` (odd but confirms to naming conventions).

Reply:
It's shouldn't be hidden.  But it should be better explained, so I added to the flag description.